### PR TITLE
Fix for truncated labels in the identity wallet

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: dev-schema/attribute/nationality.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		"denonym": {
 			"type": "string",
 			"title": "Denonym",
-			"description": "The word used to identify citizens of the country"
+			"description": "Enter the word."
```
Comment (andregoncalves) [inline dev-schema/attribute/nationality.json:17]: This doesn't look right

Comment (maycon-mello) [reply dev-schema/attribute/nationality.json:17]: Let me check with Sasha

Comment (sk91) [reply dev-schema/attribute/nationality.json:17]: it should still be a meaningful description, although a shorter one

File: dev-schema/attribute/work-place.json [hunk 2]
```diff
@@ -5,6 +5,6 @@
 	"identityAttributeRepository": "http://platform.selfkey.org/repository.json",
 	"entityType": ["individual"],
 	"title": "Workplace",
-	"description": "Please specify your current place of work. You can also state if you are un-employed or retired.",
+	"description": "Current place of work or employment status.",
```
Comment (sk91) [inline dev-schema/attribute/work-place.json:8]: work or employment seems redundant, we can leave only one of them in my opinion

File: dev-schema/attribute/nationality.json (hunk 1 above)
Comment (maycon-mello) [reply dev-schema/attribute/nationality.json]: I've updated it, please review.

File: dev-schema/attribute/work-place.json (hunk 2 above)
Comment (maycon-mello) [reply dev-schema/attribute/work-place.json]: I've updated it, please review.

**Review by andregoncalves — APPROVED** [review]
Looks good to me

**Review by sk91 — APPROVED** [review]
LGTM

**Pull request merged**